src/PerformanceTests/Tests/Categories/AmazonSQSFixture.cs

```diff
@@ -5,7 +5,7 @@ namespace Categories
     using Tests.Permutations;
     using Variables;
 
-    [TestFixture(Description = "Transports", Category = "Performance"), Explicit]
+    [TestFixture(Description = "Transports", Category = "Performance")]
```

Comment:
I removed the explicit attributes as the test adapter would ignore those tests otherwise, which doesn't seem to be what we want.

Reply:
The transportsfixture, persistencefixture and transactionmodes fixture are the tests that need to run automated. All explicit onces are for testing purposes so that developers do not need to hunt for their specific permutations.

Reply:
See previous comment